### DISCUSSION TITLE
Wrong cassandra consistency level configuration

### DISCUSSION THREAD

**Discussion by amitpriyadarshi** [Sunbird-Knowlg/Community#155] [Bugs]
@pallakartheekreddy there is a wrong cassandra consistence level configuration in learning service. In dev environment there is single node cluster but the cassandra configuration is hard coded as `QUORUM`. The configuration key is `cassandra.lp.consistency.level`. Please parameterise this so that it can be configured at adopter and environment level.

**Comment by pallakartheekreddy** [comment]
@amitpriyadarshi In the last release of Knowlg, we deprecated the learning service, and since then, we have not made any further changes to this repository.

**Comment by amitpriyadarshi** [comment]
@pallakartheekreddy do you mean all the apis are moved out of lerning-service?
In which release it is deprecated (because I checked currently sunbird-ed using relese 5.2.0 lesarning service and framework apis are still pointing to learning service only).

**Reply by pallakartheekreddy** [reply]
@amitpriyadarshi Release-5.5.0

**Reply by pallakartheekreddy** [reply]
@amitpriyadarshi Hope your query is now resolved. Please mark the thread as answered.

**Reply by pallakartheekreddy** [reply]
@amitpriyadarshi As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
